### DISCUSSION TITLE
Issues deploying Windows 2022 Server

### DISCUSSION THREAD

**Discussion by Scotte76** [vmware/packer-examples-for-vsphere#764] [Help] [answered]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.

### Project Version

latest

### VMware vSphere

VMware ESXi, 7.0.3, 21930508

### HashiCorp Packer

1.9.4

### HashiCorp Packer Plugin for VMware vSphere

packer-plugin-vsphere_v1.2.1_x5.0_linux_amd64

### Guest Operating System

WSL2 - Ubuntu 20.04.6 LTS

### Description

Hi tenthirtyam,

Firstly, I must congratulate you & your colleagues on this project, it's fantastic.

I'm trying to deploy a Windows 2022 (Option 22) it's getting to the point where it creates the VM and mounts the ISO's but then sits their waiting for an IP.

I have this on a port group/vLAN that is DHCP enabled, if I check the console it's not booting, so I checked the ISO/CD-ROM mounts, it looks like it it's mounting the Win2022 ISO but does not connect it, if I manually connect it boot from the 1st CD-ROM in the bios it starts the install then drops to a screen where it says 'No Images Available'

Please can you provide some assistance with this, I'm relatively new to the DevOps world....

Thanks

### Expected Behavior

I expect the build of the VM to complete

### Actual Behavior

As described above

### Steps to Reproduce

N/A

**Comment by tenthirtyam** [comment]
Does the datastore path and ISO file name match what's defined in the config?

**Reply by Scotte76** [reply]
Hi Ryan,

Thanks for the response

That was the first problem, I have rectified, however, it now sits at 'No Images Available', could this be related to the fact, that I am using an eval iso of Windows Server 2022?

Thanks
Scott

**Reply by tenthirtyam** [reply]
It's possible, but I have not tested with evaluation ISOs.

**Reply by nostromosec** [reply, marked as answer]
Hello everyone

I was running into the same issue last days and the eval version works by removing the "ProductKey" section from the "autounattend.pkrtpl.hcl" file. Hope this saves someone else some time.

Cheers,

**Comment by tenthirtyam** [comment]
We're making the evaluation the default but still allowing keys to be used in https://github.com/vmware-samples/packer-examples-for-vsphere/pull/844.
